sfcasts/fields.md

```diff
@@ -1,86 +1,29 @@
 
-Repeat that for last name. And then for that enabled field, I'm going to yield a buoy
-field, new enabled. Then finally I have a created at field inside of there. So I'll
-yield a date, field calling new, and then it's called my property is called created
-at so I'm just listening to same properties that I have here inside of my I entity.
-You don't see created ad only because I'm getting it from this timetable entity trait
+how do we figure out what all of the field types are? Documentation is the most obvious way. If you look on the web debug toolbar, there's a little EasyAdmin tool bar. Click into that and it will show you some basic information about what's going on. It also has a link to the document. I'll open this and keep it handy. It has a field section and down here, field types. So there's your big list of all of the different field types inside of EasyAdmin. *Or*, if you want, you can go directly to library, which is in `/vendor/easycorp/easyadmin-bundle/src/Field`. *Here* is the entire directory full of all of the different fields you can have. Let's add a few more fields inside of here.
 
-Anyways, with just this config here, if we move over and refresh beautiful. So the
-text renders like a text date knows how to handle date fields and the buoyant field
-gives us this nice little, uh, switcher. So as a challenge instead of rendering the
-first name and the last name columns, could we combine this into a single full name
-field? Well, let's try it. I'm gonna say yield full, full name inside of here. Now
-this is not a real property. If I go over to user, there is no full name property,
-but I do have inside of here a get full name method. So the question is, is it gonna
-be, be smart enough because I have full name here to read this, get full name method.
-And I bet, you know, the answer it does behind the scenes, easy admin uses the
-property accessor component of Symfony. It's the same component that's used inside of
-its form system. And it's really good at doing things like you giving it full name
-and it figuring out that it can access it via a get full name method. So back in
-configure fields,
+If you look in the `User.php` entity, you can see `$id`, `$email`, `$roles`, `$password`, `$enabled`, `$firstName`, `$lastName`, `$avatar`, and then a couple of association fields. Let's add a couple more fields for some of those. I'll `yield TextField::new('firstName')`... repeat that for `$lastName`... and then for that `$enabled` field, I'm going to `yield BooleanField::new('enabled')`. Finally, I have a `$createdAt` field inside here, so I'll `yield DateField::new()`, and then my property is called `'createdAt'`. So I'm just listing the same properties that I have here inside of my entity. You don't see `$createdAt` here only because I'm getting it from this `TimestampableEntity` trait.
```

Comment:
About whether to include `$`, best rule-of-thumb is just to be exact with what's on the screen like you did here :)

Reply:
This paragraph has a lot of first person. I'm changing to first plural (we).

For example. Instead of "I'll yield", I changed to "Let's yield" (or sometimes, as you know, I like to make more of a command like simply "Yield..."... but I don't like starting sentences with code-blocks because it looks weird to have a lowercase `yield` as the start 😛 )